### PR TITLE
DO NOT MERGE til v8.0. REP-2946 Fixing the WAR startup

### PR DESCRIPTION
**Pull request by dmnjohns**
Note that these changes _must_ be documented on the installation page of the Wiki.

Tested manually and it seems to work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,11 +126,27 @@ Repose will search for configurations in a user specified directory.
     The configuration files should be user readable and writable (chmod 600)
 
 Setting the Configuration Root Directory
-    Using web.xml, the web XML should contain:
+    If using the Valve deployment, simple pass the configuration directory to the Java process using the "-c" option.
```
Comment (wdschei) [inline README.md]: s/simple/simply/

**Comment by wdschei** [comment]
Looks good.

**Comment by wdschei** [comment]
@dmnjohns You may want to rebase this one and then @jennyvo can run the manual tests against it to prove both my fix and this one work to get the Debian WAR's back up and running.

**Comment by dmnjohns** [comment]
@wdschei Fixed the typo and rebased.

**Comment by wdschei** [comment]
Retest this please.

**Comment by dmnjohns** [comment]
The functional test container launchers still need to be fixed. I meant to do it Friday, but didn't have the time. I'm actually not sure how to get embedded Glassfish and Tomcat to provide init-params outside of the WAR itself, so I'll need to do some research there.

Of course, we could always set the cluster-id and node-id properties in the WAR we ship like we used to. The WAR web.xml should be combined with the programmatic configuration of the webapp done by the ReposeInitializer. Then Repose would work out of the box, but the user would have the option to declare their node either in the web.xml in the WAR, or in the context configuration file for their container (since it seems that containers will allow the context configuration file to override the values in the WAR's web.xml).

**Comment by dmnjohns** [comment]
retest this please

File: README.md [hunk 2]
```diff
@@ -126,11 +126,27 @@ Repose will search for configurations in a user specified directory.
...
+    If using the WAR deployment, include the following in the $CONTAINER_HOME/conf/context.xml file:
+    <Context docBase="ROOT.war">
+      <Parameter name="repose-config-directory" value="/etc/repose" override="false"/>
+      <Parameter name="repose-cluster-id" value="repose"/>
+      <Parameter name="repose-node-id" value="repose_node1"/>
+    </Context>
+
+    As an alternative, the web.xml file within the war itself can be modified to include the following:
     <context-param>
         <param-name>powerapi-config-directory</param-name>
```
Comment (wdschei) [inline README.md]: This needs updated to `repose-config-directory`.

**Comment by wdschei** [comment]
I updated this to DO NOT MERGE til v8.0 since we are changing context param name from `powerapi-config-directory` to `repose-config-directory`.

**Comment by wdschei** [comment]
Or we could just flip them back for now and fix them in v8.x.

**Comment by adrianjgeorge** [comment]
This was done because of a reported bug, does the customer need it on a 7.x release?

**Comment by wdschei** [comment]
These commits were cherry picked into the `repose-8-staging` branch.

**Pull request closed without merging**